src/lib/volume-tracker.ts

```diff
@@ -854,3 +854,4 @@ export class VolumeTracker {
 }
 
 export default new VolumeTracker(INIT_TIME);
+export { MarketMakerAddresses };
```

Comment:
It's better to put the export on the actual definition line like `export const MarketMakerAddresses = ...`.